.github/workflows/ci-obj-c.yml

```diff
@@ -15,11 +15,11 @@ concurrency:
 jobs:
   build_test:
     name: Build & Test Objective-C Wrapper
-    if: "! contains(github.event.head_commit.message, '[skip ci]')"
-    runs-on: macos-11
+    if: "! contains(github.event.head_commit.message, '[skip ci objective-c wrapper]')"
+    runs-on: macos-12
```

Comment:
Upgrading to 12, because macos-11 is not supported by the github macos runners anymore